src/styles/header.scss

```diff
@@ -0,0 +1,85 @@
+// #region header
+.header {
+  display: flex;
+  box-sizing: border-box;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  height: 60px;
+  padding-left: 50px;
+  padding-right: 50px;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+}
+
+.navi-bar {
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+}
+
+.navi-menu {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.navi-item {
+  margin: 0;
+  padding-left: 20px;
+}
+
+.navi-item:first-child {
+  padding: 0;
+}
```

Comment:
Instead of removing the padding from the first `.navi-item`, you could remove the margin from the last `.navi-item`. This would be more consistent with the idea of using margin for space between elements.